src/CommunityToolkit.Maui.Core/Primitives/Defaults/AvatarViewDefaults.shared.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace CommunityToolkit.Maui.Core;
+
+using System.ComponentModel;
+
+/// <summary>Default Values for <see cref="IAvatarView"/></summary>
+[EditorBrowsable(EditorBrowsableState.Never)]
+public static class AvatarViewDefaults
+{
+	/// <summary>Default avatar border width.</summary>
+	[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Do we need it on each property if we applied it to class?

Reply:
I mirrored what was done for `DrawingViewDefaults.shared.cs`.  More than happy to change if it's the right thing to do?